src/Cake.AzureDevOps/Pipelines/AzureDevOpsBuildsDefinitionHelper.cs

```diff
@@ -62,5 +62,64 @@ internal static class AzureDevOpsBuildsDefinitionHelper
                     .ToList();
             }
         }
+
+        /// <summary>
+        /// Updates a build definition with the new build definition settings and returns the updated build definition.
+        /// </summary>
+        /// <param name="log">The Cake log context.</param>
+        /// <param name="settings">Settings for accessing AzureDevOps.</param>
+        /// <param name="updateBuildDefinitionSettings">The settings to update the build definition.</param>
+        /// <returns>The updated Azure DevOps build definition.</returns>
+        internal static AzureDevOpsBuildDefinition UpdateBuildDefinition(
```

Comment:
To be consistent with other areas of this addin (e.g. for pull requestst) this should be implemented on the `AzureDevOpsBuildDefinition` class instead of a complete separate code flow.